### PR TITLE
Remove incremental logic from `int_netsuite2__tran_with_converted_amounts`

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_netsuite/issues/128 and https://github.com/fivetran/dbt_netsuite/issues/140

**This PR will result in the following new package version:**

v0.17.0 -- there's a model that will no longer be syncing to the warehouse due to a new ephemeral materialization

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Bug Fixes
- Adjusted the materialization of the `int_netsuite2__tran_with_converted_amounts` model **from incremental to [ephemeral](https://docs.getdbt.com/docs/build/materializations#ephemeral)**. There was a previous issue with the incremental strategy that could potentially produce duplicate records. Our team has determined that the most sensible path forward is to remove the incremental logic for this model entirely, as its performance benefits have proved marginal and an ephemeral materialization will avoid any duplication issues.
  - This is a **Breaking Change**, as `int_netsuite2__tran_with_converted_amounts` will no longer materialize in the warehouse.
 
### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [x] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

We have validation tests for all 3 end models and everything passed 
full refresh
image

incremental
image

### If you had to summarize this PR in an emoji, which would it be?
 
👻

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# dbt_netsuite v0.17.0
+[PR #153](https://github.com/fivetran/dbt_netsuite/pull/153) includes the following updates:
+
+## Bug Fixes
+- Adjusted the materialization of the `int_netsuite2__tran_with_converted_amounts` model **from incremental to [ephemeral](https://docs.getdbt.com/docs/build/materializations#ephemeral)**. There was a previous issue with the incremental strategy that could potentially produce duplicate records. Our team has determined that the most sensible path forward is to remove the incremental logic for this model entirely, as its performance benefits have proved marginal and an ephemeral materialization will avoid any duplication issues.
+  - This is a **Breaking Change**, as `int_netsuite2__tran_with_converted_amounts` will no longer materialize in the warehouse.
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:6]: Just a small suggestion to break this out a bit 
```suggestion
- Adjusted the materialization of the `int_netsuite2__tran_with_converted_amounts` model **from incremental to [ephemeral](https://docs.getdbt.com/docs/build/materializations#ephemeral)**. There was a previous issue with the incremental strategy that could potentially produce duplicate records in certain situations. 
   - Our team has determined that the most sensible path forward is to remove the incremental logic for this model entirely, as its performance benefits have proved marginal and an ephemeral materialization will avoid any duplication issues.
  - This is a **Breaking Change**, as `int_netsuite2__tran_with_converted_amounts` will no longer materialize in the warehouse.
```

File: README.md
Comment (fivetran-joemarkiewicz) [inline README.md:1]: We also need to update this section based on the updates from this PR. I imagine the changes would just be to remove the reference to the intermediate model which no longer has the incremental strategy available.
image

File: models/netsuite2/intermediate/int_netsuite2__tran_with_converted_amounts.sql
Comment (fivetran-joemarkiewicz) [inline models/netsuite2/intermediate/int_netsuite2__tran_with_converted_amounts.sql:1]: The `{% if is_incremental() %}` and other incremental logic is still included in this model. Although it won't be used per the model level config, we should still remove it.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-jamie a few change requests before approving. Also, we can also link Issue #140 to this update since it will also address that as well.

File: models/netsuite2/intermediate/int_netsuite2__tran_with_converted_amounts.sql
Comment (fivetran-jamie) [reply models/netsuite2/intermediate/int_netsuite2__tran_with_converted_amounts.sql:1]: removed the is_incremental block -- were there other spots you were thinking of?

File: README.md
Comment (fivetran-jamie) [reply README.md:1]: good catch, removed

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**